frontend/src/libs/components/button/button.tsx

```diff
@@ -5,15 +5,17 @@ type Properties = {
   type?: 'button' | 'submit';
   style?: 'primary' | 'secondary';
   isLoading?: boolean;
+  isDisabled?: boolean;
 };
 
 const Button: React.FC<Properties> = ({
   type = 'button',
   label,
   style = 'primary',
   isLoading = false,
+  isDisabled = false,
 }) => (
-  <button type={type} className={styles[style]} disabled={isLoading}>
+  <button type={type} className={styles[style]} disabled={isDisabled}>
```

Comment:
If we pass isLoading it should be disabled automatically
```suggestion
  <button type={type} className={styles[style]} disabled={isDisabled || isLoading}>
```

Reply:
done